src/charm.py

```diff
@@ -152,11 +168,10 @@ def _on_collect_unit_status(self, event: CollectStatusEvent):  # noqa C901
             logger.info("Scaling is not implemented for this charm")
             return
 
-        for relation in ["database", "fiveg_nrf", "certificates"]:
-            if not self._relation_created(relation):
-                event.add_status(BlockedStatus(f"Waiting for {relation} relation"))
-                logger.info(f"Waiting for {relation} relation")
-                return
+        if relation := self._relation_not_created():
+            event.add_status(BlockedStatus(f"Waiting for {relation} relation"))
+            logger.info(f"Waiting for {relation} relation")
```

Comment:
Please no f-strings in logging calls.
```suggestion
            logger.info("Waiting for %s relation", relation)
```

Reply:
This is already updated.